<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFieldPreferencesBase</class>
 <widget class="QDialog" name="QFieldPreferencesBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QFieldSync preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Default Import Directory</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsFileWidget" name="importDirectory"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Default Export Directory</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsFileWidget" name="exportDirectory"/>
   </item>
   <item>
    <widget class="QgsCollapsibleGroupBox" name="mGroupBox">
     <property name="title">
      <string>QFieldCloud</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Base URL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="qfieldcloud_base_url"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="qfieldcloud_username"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QgsPasswordLineEdit" name="qfieldcloud_password"/>
      </item>
      <item>
       <widget class="QPushButton" name="qfieldcloud_check">
        <property name="text">
         <string>Check connection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="qfieldcloud_check_result">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>QFieldPreferencesBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>QFieldPreferencesBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
